src/overlays/actors/ovl_En_St/z_en_st.h

```diff
@@ -6,9 +6,45 @@
 
 struct EnSt;
 
+typedef void (*EnStFunc)(struct EnSt* this, GlobalContext* globalCtx);
+
 typedef struct EnSt {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x430];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ EnStFunc actionFunc;
+    /* 0x0194 */ ColliderCylinder colCylinder[6];
+    /* 0x035C */ ColliderJntSph colSph;
+    /* 0x037C */ ColliderJntSphItem colSphItems[1];
+    /* 0x03BC */ s16 initalYaw;
+    /* 0x03BE */ s16 deathYawTarget;
+    /* 0x03C0 */ s16 groundBounces;
+    /* 0x03C2 */ s16 animFrames;
+    /* 0x03C4 */ s16 swayTimer;
+    /* 0x03C6 */ s16 setTargetYawTimer;
+    /* 0x03C8 */ s16 rotAwayTimer;
+    /* 0x03CA */ s16 rotTowardsTimer;
+    /* 0x03CC */ s16 takeDamageSpinTimer;
+    /* 0x03CE */ s16 stunTimer;
+    /* 0x03D0 */ s16 invulnerableTimer;
+    /* 0x03D2 */ s16 sfxTimer;
+    /* 0x03D4 */ s16 gaveDamageSpinTimer;
+    /* 0x03D6 */ s16 finishDeathTimer;
+    /* 0x03D8 */ s16 deathTimer;
+    /* 0x03DA */ s16 absPrevSwayAngle;
+    /* 0x03DC */ u8 playSwayFlag;
+    /* 0x03DD */ u8 teethR;
+    /* 0x03DE */ u8 teethG;
+    /* 0x03DF */ u8 teethB;
+    /* 0x03DD */ char unk_3E0[4];
+    /* 0x03E4 */ Vec3f unusedPos;
+    /* 0x03F0 */ Vec3f ceilingPos;
+    /* 0x03FC */ char unk_3FC[0x8];
+    /* 0x0404 */ s32 blurIdx;
```

Comment:
Maybe this is intended and you can just ignore this but fyi, the og name for the effect is "blure" but it may not be related to blur at all.